docs/rules/no-unsafe-finally.md

```diff
@@ -0,0 +1,99 @@
+# Disable Unsafe Statements in `finally` Block (no-unsafe-finally)
```

Comment:
disallow control flow statements in `finally` blocks (no-unsafe-finally)

The reason for all lowercase is that sometime soon the main headings will become consistent with the rules index
no-unsafe-finally: disallow control flow statements in `finally` blocks